Provis.UnitTests/Base/MapperForTests.cs

```diff
@@ -0,0 +1,16 @@
+﻿using AutoMapper;
+using Provis.Core.Helpers;
+
+namespace Provis.UnitTests.Base
+{
+    public class MapperForTests
+    {
+        public static IMapper GetMapper()
+        {
+            var mapperConfig = new MapperConfiguration(x =>
+               x.AddProfile(new ApplicationProfile()));
+
+            return mapperConfig.CreateMapper();
```

Comment:
We should add test for mapper and use mock of mapper for all other test
Reason is following the reason of failing test should be just one